forui/lib/src/foundation/focusable_properties.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+/// Properties for focusable widgets.
+class FFocusableProperties with Diagnosticable {
```

Comment:
Should properties be mixins? Feels like it can be useful to add helper methods which widgets can use.